AlicerceTests/Extensions/OptionalTests.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  OptionalTests.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+
+@testable import Alicerce
+
+final class OptionalTests: XCTestCase {
+
+    func testThen_UsingNonOptionalValue_ShouldUnwrapTheValue() {
+        let anOptionalString: String? = "😎"
+
+        anOptionalString.then { value in
+            XCTAssert(true, "🚀 \(value) unwrapped ✅")
```

Comment:
This `XCTAssert` doesn't do anything and can be converted to a simple comment 😇

To test that the `then`closure effectively ran we can simply modify `anOptionalString` inside it and assert it's value after `then` is invoked 😉